cmd/ddev/cmd/remove.go

```diff
@@ -39,17 +39,38 @@ To snapshot the database on remove, use "ddev remove --snapshot"; A snapshot is
 "ddev remove --remove-data" unless you use "ddev remove --remove-data --omit-snapshot".
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		// Prevent users from destroying everything
-		if removeAll && removeData {
```

Comment:
This is no longer an illegal combination thanks to the database snapshot feature that's a default on `--remove-data`.

Reply:
So it's fine to do `ddev rm -a --remove-data` ? Also fine to do `ddev rm -a --remove-data --omit-snapshot` That actually sounds pretty useful.

Reply:
Right. Before snapshotting was the default, `ddev rm -aR` was a great way to lose a bunch of data accidentally, but snapshots should prevent that. `ddev rm -aRO` will blow everything away, but it requires actively asking for snapshots to be skipped.